distage/distage-framework-docker/src/main/scala/izumi/distage/docker/examples/DynamoDocker.scala

```diff
@@ -9,7 +9,7 @@ object DynamoDocker extends ContainerDef {
 
   override def config: Config = {
     ContainerConfig(
-      image = "amazon/dynamodb-local:1.12.0",
+      image = "amazon/dynamodb-local:latest",
```

Comment:
Don't explicitly use `latest` because there may be unexpected upstream regressions on latest version, e.g. `postgres:latest` no longer just works without additional configuration right now:
```
[~] % docker run postgres:latest
Error: Database is uninitialized and superuser password is not specified.
       You must specify POSTGRES_PASSWORD for the superuser. Use
       "-e POSTGRES_PASSWORD=password" to set it in "docker run".

       You may also use POSTGRES_HOST_AUTH_METHOD=trust to allow all connections
       without a password. This is *not* recommended. See PostgreSQL
       documentation about "trust":
       https://www.postgresql.org/docs/current/auth-trust.html
```

So we may want to use latest known good versions, but not just blindly `latest`

Reply:
Done :+1: